Gemfile

```diff
@@ -39,6 +39,10 @@ gem "annotate"
 gem 'bootstrap', '>= 4.0.0.alpha3', '<= 4.1'
 gem "font-awesome-rails"
 
+# Slackbot framework https://github.com/slack-ruby/slack-ruby-bot
+gem 'slack-ruby-client'
+gem 'celluloid-io' # This is for the real time client, not using right now
```

Comment:
If we're not using it right now, I'd recommend either deleting the line (and making a note about the gem to yourself) or commenting it out (with a comment explaining why) so it's immediately obvious that it's not being used yet. I prefer that anything "TODO" is also marked with a link to a GitHub ticket describing what needs to be done.